Library/Homebrew/test/pathname_spec.rb

```diff
@@ -0,0 +1,308 @@
+require "tmpdir"
+require "extend/pathname"
+require "install_renamed"
+
+describe Pathname do
+  let(:src) { Pathname.new(Dir.mktmpdir) }
+  let(:dst) { Pathname.new(Dir.mktmpdir) }
+  let(:file) { src/"foo" }
+  let(:dir) { src/"bar" }
+
+  after(:each) do
+    FileUtils.rm_rf src
+    FileUtils.rm_rf dst
+  end
+
+  describe DiskUsageExtension do
+    before(:each) do
+      FileUtils.mkdir_p dir/"a-directory"
```

Comment:
It was imported in the previous test, but I think explicitly using `FileUtils` makes it more obvious where this is coming from, because there are already a bunch of similar methods for `Pathname`.